lms/djangoapps/ora_staff_grader/serializers.py

```diff
@@ -159,6 +161,56 @@ class Meta:
             "gradeStatus",
             "lockStatus",
             "score",
+            "email",
+            "fullname",
+        ]
+        read_only_fields = fields
```

Comment:
Could you remind me why we are adding this specific line?

Reply:
This is part of the changes to the `initialize` endpoint to include the `email` and `fullname` fields in each submission.